command/update/command.go

```diff
@@ -56,8 +54,8 @@ func New(config Config) (*Command, error) {
 
 	newCommand.cobraCommand = &cobra.Command{
 		Use:   "update",
-		Short: "Update Kubernetes endpoints based on given configuration.",
-		Long:  "Update Kubernetes endpoints based on given configuration.",
+		Short: "Update annotation on KVM pod based on given configuration.",
+		Long:  "Update annotation on KVM pod based on given configuration.",
```

Comment:
Nitpick `annotations`. It is about multiple annotations potentially so lets use plural.

Reply:
fixed